app/api/events.py

```diff
@@ -451,7 +439,7 @@ def get_id(view_kwargs):
             view_kwargs['id'] = None
 
     if view_kwargs.get('track_id') is not None:
-        track = safe_query(db, Track, 'id', view_kwargs['track_id'], 'track_id')
+        track = safe_query(Track, 'id', view_kwargs['track_id'], 'track_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)